codeSnippets/snippets/tutorial-server-routing-and-requests/src/main/kotlin/model/TaskRepository.kt

```diff
@@ -0,0 +1,24 @@
+package model
+
+object TaskRepository {
+    private val tasks = mutableListOf(
+        Task("cleaning", "Clean the house", Priority.Low),
+        Task("gardening", "Mow the lawn", Priority.Medium),
+        Task("shopping", "Buy the groceries", Priority.High),
+        Task("painting", "Paint the fence", Priority.Medium)
+    )
+
+    fun allTasks(): List<Task> = tasks
+    fun tasksByPriority(priority: Priority) = tasks.filter {
```

Comment:
I would keep an empty line between methods for readability; you can invoke autoformat in IDEA or use detekt plugin, they can fix it